definitions/firefox_desktop.toml

```diff
@@ -2051,6 +2089,19 @@ description = "Retention Outcomes of Desktop New Profiles Over their First 112 D
 friendly_name = "Desktop Cohorts Daily Retention"
 client_id_column = "NULL"
 
+[data_sources.desktop_engagement_view]
+from_expression = "moz-fx-data-shared-prod.telemetry.desktop_engagement"
```

Comment:
Should we add the is_desktop filter to this one for consistency with mobile?

Reply:
Good call! I made those changes.